source/frontend/src/features/projects/reports/ProjectExportContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { useCallback } from 'react';
+
+import { useAcquisitionProvider } from '@/hooks/repositories/useAcquisitionProvider';
+import { useProjectProvider } from '@/hooks/repositories/useProjectProvider';
+import { Api_ExportProjectFilter } from '@/models/api/ProjectFilter';
+
+import { IProjectExportFormProps } from './ProjectExportForm';
+
+export interface ISideProjectContainerProps {
+  View: React.FunctionComponent<React.PropsWithChildren<IProjectExportFormProps>>;
+}
+
+export const SideProjectContainer: React.FunctionComponent<ISideProjectContainerProps> = ({
```

Comment:
nit: mild preference to have class name match filename ProjectExportContainer vs SideProjectContainer